rom/src/rom.rs

```diff
@@ -249,9 +249,58 @@ impl Soc {
         }
 
         // TODO: vendor-specific fuses when those are supported
-        // TODO: load ECC Revocation CSRs.
-        // TODO: load LMS Revocation CSRs.
-        // TODO: load MLDSA Revocation CSRs.
+        // Load Owner ECC/LMS/MLDSA revocation CSRs.
+        // ECC Revocation.
+        let vendor_ecc_revocation = u32::from_le_bytes(
+            fuses
+                .cptra_core_ecc_revocation_0()
+                .try_into()
+                .unwrap_or_else(|_| fatal_error(McuError::ROM_SOC_KEY_MANIFEST_PK_HASH_LEN_MISMATCH)),
```

Comment:
Can you make unique error codes for these? It would be really helpful debugging if something goes wrong

Reply:
Confirmed, no panic.

Reply:
Lets just use `unwrap()` for these then